src/main/java/frc/robot/subsystems/led/LED.java

```diff
@@ -49,17 +48,17 @@ public void setSolidColorNumber(Color color, int length) {
     m_LEDStrip.setData(m_LEDStripBuffer);
   }
 
-  public Command toggleColor() {
-    return Commands.run(() -> {
-      if (currentToggle == true) {
-        setSolidColor(m_colors[1]);
-        currentToggle = false;
-      } else {
-        setSolidColor(m_colors[0]);
-        currentToggle = true;
-      }
-    });
-  }
+  // public Command toggleColor() {
+  //   return Commands.run(() -> {
+  //     if (currentToggle == true) {
+  //       setSolidColor(m_colors[1]);
+  //       currentToggle = false;
+  //     } else {
+  //       setSolidColor(m_colors[0]);
+  //       currentToggle = true;
+  //     }
+  //   });
+  // }
```

Comment:
Unless you intend to use something in the future, you should just remove it instead of commenting it out. It can always be found in the git history 🙂